scripts/js/commands/checkMarkdown.ts

```diff
@@ -56,9 +56,10 @@ async function main() {
         ? new Set<string>()
         : collectHeadingTitleMismatch(tree, metadata);
 
-    const metadataErrors = METADATA_ALLOWLIST.has(file) || file.startsWith("docs/api")
-      ? new Set<string>()
-      : checkMetadata(metadata);
+    const metadataErrors =
+      METADATA_ALLOWLIST.has(file) || file.startsWith("docs/api")
```

Comment:
@GantaRoja this is the wrong fix. We do want to check metadata for the API docs. We only don't want to check specifically the line length. Look at the original code in `checkMetadata.ts`: you need to pass the filepath to the function `checkMetadata`